.github/workflows/haskell-ci.yml

```diff
@@ -228,6 +249,9 @@ jobs:
           if [ $((HCNUMVER >= 80200)) -ne 0 ] ; then echo "    ghc-options: -Werror=missing-methods" >> cabal.project ; fi
           cat >> cabal.project <<EOF
           EOF
+          if $HEADHACKAGE; then
+          echo "allow-newer: $($HCPKG list --simple-output | sed -E 's/([a-zA-Z-]+)-[0-9.]+/*:\1,/g')" >> cabal.project
```

Comment:
note: you won't test whether you relaxations are good enough for GHC-9.4.1, as this line is `--allow-newer` everything in `ghc-pkg` boot database.

Reply:
Thanks for the heads up!

I see, `HEADHACKAGE` will be set to `true` automatically by Haskell-CI for pre-release versions of GHC.
This makes the `head-hackage: > 9.3` obsolete which I thought I have to put into `cabal.haskell-ci`, e.g.: https://github.com/haskell-pkg-janitors/haskell-src/blob/a5c36e4c76b0f7b97b19f7c2d4a97738cc5f821a/cabal.haskell-ci#L2